cookie.js

```diff
@@ -61,18 +53,18 @@ const fieldContentRegExp = /^[\u0009\u0020-\u007e\u0080-\u00ff]+$/ // eslint-dis
  * The object has the various cookies as keys(names) => values
  *
  * @param {string} str
- * @param {object} [options]
+ * @param {object} opt
  * @return {object}
  * @public
  */
 
-function parse (str, options) {
+function parse (str, opt) {
```

Comment:
Is there a reason for renaming the parameter? Doing so seems to only be obscuring the real changes.